panda/src/collide/collisionHeightfield.h

```diff
@@ -0,0 +1,159 @@
+#ifndef COLLISIONHEIGHTFIELD_H
+#define COLLISIONHEIGHTFIELD_H
+
+#include "pandabase.h"
+#include "collisionSolid.h"
+#include "pnmImage.h"
+
+using std::vector;
```

Comment:
I believe we should be avoiding `using` statements in header files (and maybe in general) since it can lead to polluted namespaces. However, I will defer to @rdb's opinion on this.

Reply:
I agree with this; this should not be in the header file.

Reply:
OK, removed it :+1: